packages/next/server/lib/find-page-file.ts

```diff
@@ -60,3 +60,11 @@ export async function findPageFile(
 
   return existingPath
 }
+
+// Determine if the file is leaf node page file under layouts,
+// They filename should start with 'page', it can be either shared,
+// client or server components with allowed page file extension.
+// e.g. page.js, page.server.js, page.client.tsx, etc.
```

Comment:
```suggestion
// Determine if the file is leaf node page file under layouts,
// The filename should start with 'page', it can be either shared,
// client, or server components with allowed page file extension.
// e.g. page.js, page.server.js, page.client.tsx, etc.
```